.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 /.php-version
 /build/
 /vendor/
+/*.code-workspace
```

Comment:
Also, this should probably stay on your development machine.

Reply:
Agreed